netcode.c

```diff
@@ -3491,12 +3491,13 @@ struct netcode_server_t
     int running;
     int max_clients;
     int num_connected_clients;
-    int max_clients_per_match;									/* Max clients per match */
+    int max_clients_per_match;											/* Max clients per match */
```

Comment:
what's with the tabs and weird spacing here?  In TS at least, we put comments on the line before and try to avoid having them on the same line...  This takes that to another level.  Not to mention I don't think this comment adds any value, it just repeats the variable without the underscores.

Reply:
Its not weird in my IDE, I have no clue why git is being weird about it.  But I can remove it.

Reply:
NVM my vim says its off but not the ide.